CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.0.4] - 2020-01-16
```

Comment:
This should be `4.1.0` since this is new functionality, not a bug release.

Reply:
👍